appcues/src/main/java/com/appcues/ui/modal/DialogModal.kt

```diff
@@ -33,48 +43,119 @@ private val MAX_WIDTH_EXPANDED_DP = 560.dp
 private val MAX_HEIGHT_COMPACT_DP = Dp.Unspecified
 private val MAX_HEIGHT_MEDIUM_DP = 800.dp
 private val MAX_HEIGHT_EXPANDED_DP = 900.dp
-private const val SCREEN_PADDING = 0.05
+
+private val SCREEN_HORIZONTAL_PADDING = 12.dp
+private val SCREEN_VERTICAL_PADDING = 24.dp
+
+internal data class DialogTransition(
```

Comment:
up to you if it looks more organized this way, but since all information is coming from style you could build this object inside of DialogModal since this object does not matter too much to the other presentation styles.

Reply:
right now, this information is coming in through `ModalTrait`, since it is in the trait config (outside of style) and needs to be passed through into the `DialogModal` composition. So that is why I made this extra data class to capture and pass the info through here. Maybe I am missing another point on simplification though?

Reply:
you are right I for some reason thought this was coming from Style.